modules/tile-converter/.gitignore

```diff
@@ -1 +1,2 @@
-./data
+data
```

Comment:
Nit: Maybe choose another name to avoid conflicting with `test/data`. `installed-data`?